front/src/common/AddNewCard.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ReactNode } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { FaPlus } from 'react-icons/fa';
+
+import useUserRoleCheck from './authorization/hooks/useUserRoleCheck';
+import { REQUIRED_USER_ROLES_FOR } from './authorization/roleBaseAccessControl';
+import { useModal } from './BootstrapSNCF/ModalSNCF';
+
+type AddNewCardProps = {
+  translationNamespaces?: string[] | string;
+  testId: string;
+  className: string;
+  modalComponent: ReactNode;
+  legendTranslationKey: string;
+};
+
+const AddNewCard = ({
+  translationNamespaces,
+  testId,
+  className,
+  modalComponent,
+  legendTranslationKey,
+}: AddNewCardProps) => {
+  const { t } = useTranslation(translationNamespaces);
+  const { openModal } = useModal();
+
+  const newProjectStudyScenarioAllowed = useUserRoleCheck(
+    REQUIRED_USER_ROLES_FOR.FEATURES.CREATE_NEW_PROJECT_STUDY_SCENARIO
+  );
+
+  return (
+    <div
+      data-testid={testId}
+      className={`${className} ${!newProjectStudyScenarioAllowed ? 'aria-disabled' : ''}`}
```

Comment:
`aria-disabled` is not a class, it's an attribute. It's an accessibility feature to mark an element as disabled, useful for screen readers and similar. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled